src/tests/pages.test.js

```diff
@@ -0,0 +1,41 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../api/store';
+import ContinentDetails from '../features/ContinentDetails';
+import CountryDetails from '../features/CountryDetails';
+import Dashboard from '../features/Dashboard';
+
+describe('UI Tests', () => {
+  it('should render Dashboard', () => {
+    const tree = render(
+      <BrowserRouter>
+        <Dashboard />
+      </BrowserRouter>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render Continent Details', () => {
+    const tree = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <ContinentDetails />
+        </BrowserRouter>
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should render Country Details', () => {
+    const tree = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <CountryDetails />
+        </BrowserRouter>
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Nice job writing snapshot tests for the UI 👍. You are also required to write integration tests for your react components and redux code making use of the react-testing-library. Kindly implement that Cheers 👍.

![image](https://github.com/til2to/country-pulse/assets/60618877/aa943b1d-95b2-4b2d-80f4-17c5e9d2394d)